contributing.rst

```diff
@@ -619,9 +619,9 @@ In order to facilitate efficient :ref:`cross-referencing`, sections should be la
 
   Lorem ipsum content of section blah blah.
 
-The section label should usually be a sluggified version of the section title.
+The section label is a sluggified version of the section title.
```

Comment:
But it isn't always.

Reply:
If there isn't a reason it shouldn't be then shouldn't we push the narrative that it is? Otherwise it might just creates confusion in the mind of the reader - or at least that was my thinking.

Reply:
I see your point. But here's my thinking:

 - I don't want someone coming through and "helping" by doing line edits to correct existing section labels.
 - Sometimes you want a short, clear label but it makes sense to have a longer, descriptive, human-readable title.
 - Sometimes you change the human readable title. But the label should typically not be changed, as it might have cross references.

You might be right about this, though. 
But a line edit isn't the place to have this discussion.

Add a note to the Style Guide issue, that this is something we should hash out. In the mean time, leave this instruction in place.

Reply:
Thank you for your detailed reply. I will do as you suggested and add these points as a comment to the Style Guide issue.